pkg/apis/core/validation/events_test.go

```diff
@@ -254,7 +254,7 @@ func TestValidateEventForNewV1beta1Events(t *testing.T) {
 			Event: &core.Event{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      "test",
-					Namespace: metav1.NamespaceSystem,
+					Namespace: metav1.NamespaceDefault,
```

Comment:
Revert the changes to this file

Reply:
As you and @wojtek-t  discussed, we decided to change the validation function. Isn't it ?

Reply:
as noted in https://github.com/kubernetes/kubernetes/pull/100125#issuecomment-912823869, I think we have to *relax* API validation to allow either the `kube-system` or `default` namespaces for events for cluster-scoped objects.

That means this unit test should not change (since NamespaceSystem would still be allowed), but additional tests should be added to ensure NamespaceDefault is *also* allowed

Reply:
> As you and @wojtek-t discussed, we decided to change the validation function. Isn't it ?

yes, in the API validation, where we currently only allow NamespaceSystem, we should allow NamespaceSystem or NamespaceDefault